lnbits/decorators.py

```diff
@@ -134,9 +136,10 @@ async def get_key_type(
     token = api_key_header if api_key_header else api_key_query
 
     try:
-        checker = WalletAdminKeyChecker(api_key=token)
-        await checker.__call__(r)
-        wallet = WalletTypeInfo(0, checker.wallet)
+        admin_checker = WalletAdminKeyChecker(api_key=token)
+        await admin_checker.__call__(r)
+        wallet = WalletTypeInfo(0, admin_checker.wallet)
+        assert wallet.wallet is not None
```

Comment:
Why rename variable here? Because of same name below? Is that a problem?

Reply:
its similar to the user wallet, mypy doesnt like if you use the same variable name with different types, in this case checker first was WalletAdminKeyChecker and the it is used later with type WalletInvoiceKeyChecker

Reply:
Gotcha!